starfish/pipeline/features/_validated_table.py

```diff
@@ -2,6 +2,9 @@
 
 
 class ValidatedTable:
+
+    required_fields = NotImplemented
```

Comment:
I don't think this is how NotImplemented is supposed to be used as per the docs but it's not a dealbreaker.

Reply:
That's actually an interesting one -- I needed to define that for mypy to let me define load in the base class, since it requires that object. should I... make it a `property` and have the property return `NotImplementedError`? I don't think that would harm the inheritance pattern.

Reply:
I don't have a great solution for this.  I just slapped an annotation to bypass the check in my PR.

Reply:
https://github.com/spacetx/starfish/pull/227/files#diff-9bdf66c5007c8d6623fc4e46338b6b0aR67

Reply:
I tried making it a property (see #230) but mypy got angry with me for reasons related to the property.